lib/bot.rb

```diff
@@ -0,0 +1,19 @@
+class Bot
+  def initialize(client: nil)
+    @client = client || Slack::Web::Client.new
+  end
+
+  def send(message, user_id)
+    Logger.info("BOT RESPONSE: #{message}, to: #{user_id}")
+
+    client.chat_postMessage(
+      as_user: 'true',
+      channel: user_id,
+      text: message
+    )
+  end
+
+  private
+
+  attr_reader :client
```

Comment:
Does it worth to have that in order to avoid to `@` line 9?

Reply:
I generally prefer to use `attr_reader` because it quickly gets messy with `@` everywhere, but I am not religious about it.

Reply:
Yeah, I guess it could be good to stick to one of them in order to remain consistent, either way, I'm happy with it 🙏